src/qt/veil.cpp

```diff
@@ -146,16 +146,12 @@ static void initTranslations(QTranslator &qtTranslatorBase, QTranslator &qtTrans
 /* qDebug() message handler --> debug.log */
 void DebugMessageHandler(QtMsgType type, const QMessageLogContext& context, const QString &msg)
 {
-#ifndef QT_MESSAGELOGCONTEXT
-   Q_UNUSED(context);
+    Q_UNUSED(context);
     if (type == QtDebugMsg) {
         LogPrint(BCLog::QT, "GUI: %s\n", msg.toStdString());
     } else {
         LogPrintf("GUI: %s\n", msg.toStdString());
     }
-#else // QT_MESSAGELOGCONTEXT needs to be added to the DEFS line of the Makefile
-    LogPrintf("GUI: %s:%d %s\n", context.file, context.line, msg.toStdString());
-#endif
```

Comment:
Let's comment out the relevant parts instead of deleting them; Add a comment also that says something like "can be used for debugging but may need changes for MacOS debugging"

Reply:
Sure, I don't really care.  
How about :  "Please don't enable this publicly. It may leak information, and we've been bit by a nasty crash where the context is not available in non-debug builds."

Reply:
I'm still unsure where that QT_MESSAGELOGCONTEXT is coming from, probing around.

Reply:
It's set by the Qt headers depending on how Qt was built.

Reply:
okay.  yea, just a message so it's clear why it's commented out, but there for anyone finding it by chasing through the debug logging of Qt.

Since it's dependent on the building style, yep.  Removing it is the right approach rather than changing the building style to remove it.  Perhaps we can re-visit later when we figure out exactly what's needed to clear up the crashes... e.g. `#ifndef (QT_MESSAGECONTEXT || DEBUG) `or whatever, and put it back in.  But for now this is good enough.